tests/components/nice_go/test_diagnostics.py

```diff
@@ -14,6 +15,7 @@
 from tests.typing import ClientSessionGenerator
 
 
+@pytest.mark.freeze_time("2024-08-27")
```

Comment:
Not sure this is still needed, as the fixture now holds the current time that is freezed, we should not leave the test in the past

```suggestion
```